### ISSUE TITLE
Error when inserting template with INPUT tag

### ISSUE THREAD

**Issue by hiasr** [thundernest/quicktext#348]
I created multiple templates and everything worked fine until adding an input tag.
It is possible to create and save the template, but when trying to insert it, I get the following error: 

```
Uncaught (in promise) TypeError: Components.classes['@mozilla.org/embedcomp/prompt-service;1'] is undefined
    process_input chrome://quicktext/content/modules/wzQuicktextVar.jsm:663
    get_input chrome://quicktext/content/modules/wzQuicktextVar.jsm:346
    parseText chrome://quicktext/content/modules/wzQuicktextVar.jsm:176
    parse chrome://quicktext/content/modules/wzQuicktextVar.jsm:117
    insertSubject chrome://quicktext/content/quicktext.js:521
    insertTemplate chrome://quicktext/content/quicktext.js:362
    updateGUI chrome://quicktext/content/quicktext.js:278
```
I am currently running thunderbird beta 114.0b5 on Fedora, I tried this both on plugin version v5.3 and v5.5.
Thank you in advance!

**Comment by hiasr** [comment]
I tried to look into this, and it seems that `'@mozilla.org/embedcomp/prompt-service;1'` is not a part anymore of thunderbird. I looked for an easy replacement but couldn't really find one. This means that the input tag will stop working in one of the following stable versions.

**Comment by SamuelPlentz** [comment]
jobisoft confirmed that the old INPUT Form gets deprecated in #281:

> The underlying xul element has been deprecated. We need to convert to html.

**Comment by jobisoft** [comment]
I was wrong. It is still there, but is now only behind `Services.prompt.*`
Fixed here: 92bd2bf48fa17ede2a8042b3cfb02f582cd4e01e

**Comment by jobisoft** [comment]
Please verify:
https://github.com/jobisoft/quicktext/releases/tag/v5.6

**Comment by hiasr** [comment]
Solved! Thank you!

**Comment by SamuelPlentz** [comment]
I can confirm, that this is fixed in version 5.6.
